tests/test_cli.py

```diff
@@ -103,6 +103,20 @@ def test_run_simple(runner, project):
     assert '.renku/workflow/' in result.output
 
 
+def test_run_many_args(client, run):
+    """Test a renku run command which implicitly relies on many inputs."""
+
+    os.mkdir('files')
+    output = 'output.txt'
+    for i in range(5000):
```

Comment:
I see a potential edge case here. Can we test with a number which is not dividable by 1000? 😄

Reply:
Sure - how about 5003 (Sophie Germain prime number) acceptable? 😃

Reply:
Yes! Perfect!